pkg/graph/schema.resolvers.go

```diff
@@ -39,6 +39,8 @@ func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *model
 		//
 		// We will either update the records in the database with the results OR implement
 		// another mechanism for doing that as a background job so that we don't need to do it here.
+		// Furthermore, locally this will always return "", since we are not interfacing with the
```

Comment:
We should deal with this, but I don't know how. Since we're not using the real S3 when we test locally, our files that we "upload" never pass the virus scan. When we view files on the front end, we won't see the Edit/Delete buttons, we'll just see Virus Scanning In Progress.